devtools/common/src/main/java/io/quarkus/generators/rest/BasicRestProjectGenerator.java

```diff
@@ -76,13 +77,19 @@ private boolean initProject() throws IOException {
 
             srcMainPath = writer.mkdirs(type.getSrcDir());
             testMainPath = writer.mkdirs(type.getTestSrcDir());
+            // for gradle we want to place the native tests in under 'src/nativeTest/java'
```

Comment:
camel case for paths is not very appealing, I think I would prefer `native-test`.

Reply:
Sure, let's do that

Reply:
Fixed